js/bibox.js

```diff
@@ -2211,34 +2211,41 @@ module.exports = class bibox extends Exchange {
                 if (method === 'GET') {
                     url += '?' + this.urlencode (params);
                 } else {
-                    if (json_params !== '{}') {
+                    if (jsonParams !== '{}') {
                         body = params;
                     }
                 }
             } else if (v4) {
                 let strToSign = '';
+                const sortedParams = {};
+                const keys = Object.keys (params);
```

Comment:
@samgermain you don't need to do this manually, you can use `this.keysort(params)` here